src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -148,6 +149,9 @@ Author(s):
     X(bool, RainbowFrame, "experimental.rainbowFrame", false)                                                                      \
     X(bool, UseMica, "useMica", false)
 
+#define MTSM_THEME_SETTINGS_SETTINGS(X) \

```

Comment:
Does this macro get called anywhere?

Reply:
![image](https://github.com/microsoft/terminal/assets/18356694/34819bbe-78f4-454b-8451-b1283162d037)